server/index.js

```diff
@@ -0,0 +1,63 @@
+const express = require('express');
+const morgan = require('morgan');
+const path = require('path');
+
+const app = express();
+const httpProxy = require('http-proxy');
+const proxy = httpProxy.createProxyServer();
+
+const imageMain = 'http://127.0.0.1:3000';
+const experiences = 'http://127.0.0.1:3636';
+const bestNearby = 'http://127.0.0.1:3003';
+const reviews = 'http://127.0.0.1:3004';
+const proxyPort = 3006;
+
+const publicDir = path.resolve(__dirname, '..', 'client', 'public');
+
+app.use(morgan('dev'));
+app.use('/:productId', express.static(publicDir));
+
+// ==============================IMAGE MAIN=============================
```

Comment:
So great Chelsea! So clear. 

The only comment i can make is if we can standardize the end points for each component.

Reply:
Agreed! After everyone's got something working, we should touch base as a group and agree on what to call everything. I had to make a temporary change in my copy of everyone's repo to get endpoints to match up to get this working